test/enron.cc

```diff
@@ -0,0 +1,94 @@
+#include <iostream>
+#include <set>
+#include <string>
+#include <fstream>
+
+#include <cedar_config.h>
+#include <cedar.h>
+
+using Trie = cedar::da<int>;
+
```

Comment:
we should move this file to a benchmark dir

And could you add here or in some README.md,  how you ran the benchmark 

On which dataset
"find . -type f" > file list
etc